docs/docs/workers.md

```diff
@@ -116,18 +116,14 @@ A simple implementation example:
 
 ```python
 #!/usr/bin/env python
-import sys
 from rq import Connection, Worker
 
 # Preload libraries
 import library_that_you_want_preloaded
 
-# Provide queue names to listen to as arguments to this script,
-# similar to rq worker
+# Provide the worker with the list of queues (str) to listen to.
 with Connection():
```

Comment:
Can we eliminate this pattern from docs altogether? I want to slowly deprecate this context manager, would prefer explicitly passing in Redis connection to `Queue` and `Worker` objects.

Reply:
I agree with removing the context manager, we already have the connection being injected, I think it's a lot cleaner to just have it as a parameter handled by the user.

I would keep the example (preloading library) and change the context manager to a explicit connection, what do you think?

Reply:
Yes, this sounds good. Thanks!